call.go

```diff
@@ -143,6 +166,9 @@ func Invoke(ctx context.Context, method string, args, reply interface{}, cc *Cli
 			}
 			return toRPCErr(err)
 		}
+		if enableTracing {
+			c.traceInfo.tr.LazyLog(payload{args}, true)
```

Comment:
Any security concerns exposing args or payload in traces by default?
Will this keep large args and payloads pinned in memory longer than users expect?